pages/friends.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import FriendsList from "./components/FriendsList";
+import Form from "./components/Form";
+import React from "react";
+import { useState } from "react";
+import { uid } from "uid";
+
+export default function Friends() {
+  const [friends, setFriends] = useState([]);
+
+  function handleAdd(newFriend) {
+    console.log(friends);
```

Comment:
Remove logs.

Reply:
ok, removed.

Reply:
i removed already.
should i make a new code-review?